src/app/components/drawer/Projects/ProjectsComponent.js

```diff
@@ -21,7 +21,7 @@ import FileDownloadIcon from '../../../icons/file_download.svg';
 import InboxIcon from '../../../icons/inbox.svg';
 import LightbulbIcon from '../../../icons/lightbulb.svg';
 import PersonIcon from '../../../icons/person.svg';
-import PublishedIcon from '../../../icons/playlist_add_check.svg';
+import PublishedIcon from '../../../icons/fact_check.svg';
```

Comment:
FYI: I just noticed an inconsistency with icons while i was doing this...plus then its not ONLY a subtracting code PR